openage/opus/setup.py

```diff
@@ -0,0 +1,14 @@
+# this file should be removed in the future.
+# asdf
+from distutils.core import setup
+from distutils.extension import Extension
+from Cython.Build import cythonize
+
+setup(
+    name = 'opusencoder',
+    ext_modules = cythonize([
+        Extension("opusenc",["opusenc.pyx"],
+                   libraries=["ogg","opus"])
+        ])
+)
+
```

Comment:
okay, `set(PYEXT_LINK_LIBRARY ogg opus)` is the right way to go. but I don't know how it linked to opus without that in CMakeLists.txt